src/clarity/buttons/_buttons.clarity.scss

```diff
@@ -265,13 +264,23 @@ $clr-button-appearance-map: (
         }
 
         &.btn-sm {
-            @include clr-getStylesFromMap($clr-button-appearance-map,standard);
+            @include clr-getStylesFromMap($clr-button-appearance-map, standard);
+
+            clr-icon {
+                width: 12px;
+                height: 12px;
+            }
         }
 
         &.btn-block {
             display: block;
             width: 100%;
             max-width: 100%;
         }
+
+        clr-icon {
+            vertical-align: text-bottom;
```

Comment:
Does this alignment work on all browsers? I have noticed that the `vertical-align` & `margin` combo works a bit differently in IE and Firefox. Please confirm? If possible please paste screenshots.